data.sql

```diff
@@ -74,3 +74,73 @@ WHERE name = 'Charmander' or name = 'Squirtle' or name = 'Blossom';
 UPDATE animals
 SET owner_id = 5
 WHERE name = 'Angemon' or name = 'Boarmon';
+
+
+-- Insert the following data for vets:
+  -- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23');
+
+-- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Maisy Smith', 26, '2019-01-17');
+
+-- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Stephanie Mendez', 64, '1981-05-04');
+
+-- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert the following data for specialties:
+  -- Vet William Tatcher is specialized in Pokemon.
+  -- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
+  -- Vet Jack Harkness is specialized in Digimon.
+INSERT INTO specializations (species_id, vet_id) VALUES (1, 1);
+INSERT INTO specializations (species_id, vet_id) VALUES (1, 3);
+INSERT INTO specializations (species_id, vet_id) VALUES (2, 3);
+INSERT INTO specializations (species_id, vet_id) VALUES (2, 2);
```

Comment:
- [ ] You are using the ids here and in many cases in your queries like just numbers. For a foolproof search in the queries, please use the right query to get the id automatically. Like this:

>  INSERT INTO specializations (species_id, vet_id) VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (...)